tech-committee-charter.md

```diff
@@ -61,6 +61,34 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
+
+* *Escalation Sponsorship*: The TC sponsor will be a primary means of escalation of cross-project concerns from the SIG. This communication is primarily
+  offline, e.g. via CNCF Slack or issue communication. The TC sponsor will participate in [GC check-ins](gc-check-ins.md) with SIG leadership.
+  Additionally, the TC will aide the SIG in developing road maps during this check-in, and aligning to overall OpenTelemetry goals.
+* *Guiding Sponsorship*: The TC sponsor is an active participant of the SIG. The sponsor will help direct the SIGs efforts to match the technical goals of OpenTelemetry.
+* *Leading Sponsorship*: The TC sponsor is an active member of SIG leadership, driving the direct success of the project or proposal.
+
+All SIGs within OpenTelemetry will be required to have some level of sponsorship requirement (at least *Escalation*). Higher level sponsorship automatically fulfills
+lower level sponsorship requirements, e.g. *Guiding* sponsorship fulfills an *Escalation* sponsorship requirement, as *Leading* sponsorship would fulfill an
+*Escalation* sponsorship requirement. Sponsorship is not exclusive. More than one TC member may sponsor a SIG.
+
+As part of their active leadership, every TC member will be required to sponsor SIGs across OpenTelemetry.
+
+Every TC member is required to provide a minimum of one (1) *Escalation* sponsorship. All SIGs requiring escalation sponsorship will be divided across the TC.
+
+Every TC member is required to provide a minimum of two (2) *Guiding* or *Leading* sponsorships.
```

Comment:
To clarify that, this would also allow a TC member to only be Guiding Sponsor for 2 projects, and not leading a single one?

Reply:
Yes.